integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5872,6 +5872,42 @@ msgstr "Seite neu laden"
 msgid "Continue reading"
 msgstr "Weiterlesen"
 
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Ask Integreat Statistics"
+msgstr "Frag Integreat Statistiken"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Chats in last 6 hours"
+msgstr "Unterhaltungen in Letzten 6 Stunden"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Open Zammad"
+msgstr "Zammad Öffnen"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Total Chats"
+msgstr "Gesamtzahl Unterhaltungen"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Used Budget"
+msgstr "Verbrauchtes Budget"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Chats Last Month"
+msgstr "Unterhaltungen im Letzten Monat"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Used Budget Last Month"
+msgstr "Genutztes Übersetzungsbudget im Letzten Monat"
+
+#: cms/templates/dashboard/_public_chat_stats.html
+msgid "Chats Previous Month"
+msgstr "Unterhaltungen im Vorletzten Monat"
```

Comment:
```suggestion
msgstr "Unterhaltungen im vorletzten Monat"
```

Reply:
Indeed, initially I was thinking about the text as headlines. But they're not anymore so the capitalization is no longer suitable.